src/presentation/test/integration/registration/generalPartner/address/confirm-general-partner-correspondence-address.test.ts

```diff
@@ -72,6 +76,21 @@ describe("Confirm General Partner Correspondence Address Page", () => {
       expect(res.text).toContain("England");
       expect(res.text).toContain("ST6 3LJ");
     });
+
+    it.each([
+      ["overseas", getUrl(ENTER_GENERAL_PARTNER_CORRESPONDENCE_ADDRESS_URL)],
+      ["unitedKingdom", getUrl(POSTCODE_GENERAL_PARTNER_CORRESPONDENCE_ADDRESS_URL)]
+    ])("should have the correct back link", async (territory, backLink) => {
+      appDevDependencies.cacheRepository.feedCache({
+        [appDevDependencies.transactionGateway.transactionId]: {
+          sa_territory_choice: territory
```

Comment:
Perhaps it doesn't matter in the test, but should this key reference `ura_territory_choice_limited_partner`?

Reply:
Ignore, just seen this is for the general partner.